internal/e2e/sdk_client_test.go

```diff
@@ -286,7 +288,7 @@ func (c *sdkClient) waitUntilLive(t require.TestingT) {
 	for err != nil {
 		resp, _, err = c.getReadClient().MetadataApi.IsReady(c.requestCtx()).Execute()
 	}
-	require.Equal(t, "ok", resp.Status)
+	require.Equal(t, "SERVING", resp.Status)
```

Comment:
Yes this is breaking, but only if you look at the content and not the code. Working around this would be hacky, as the "SERVING" response is standard in gRPC land.